app/oxalis/security/Environment.scala

```diff
@@ -19,7 +19,7 @@ class WebknossosEnvironment @Inject()(configuration: Configuration)(implicit val
   val cookieSettings = configuration.underlying.as[CookieAuthenticatorSettings]("silhouette.cookieAuthenticator")
   val tokenSettings = configuration.underlying.as[BearerTokenAuthenticatorSettings]("silhouette.tokenAuthenticator")
   val fingerprintGenerator = new DefaultFingerprintGenerator(false)
-  val idGenerator = new SecureRandomIDGenerator()
```

Comment:
There is the unused (I thinks so at least) import of `SecureRandomIDGenerator` left now